### ISSUE TITLE
New version of Spine don't have configure file

### ISSUE THREAD

**Issue by mariusmotea** [scline/docker-cacti#49]
Hi,

I notice that new builds of spine don't have configure file so upgrades are applied just to Cacti. In Cacti dashboard error is displayed because cacti version is newer than spine.

To fix it i need to execute `autoreconf -fvi`  and `autoconf` in order to generate the configure file.

**Comment by scline** [comment]
Thanks for the report, Ill look into what configuration file it requires and next steps.

**Comment by scline** [comment]
Looks like there are a few things to correct,
```
docker exec 88271b9f144d ./upgrade.sh
2019-09-06_11:47 [Upgrade] Prepping workspace for restore.
2019-09-06_11:47 [Upgrade] Downloading Cacti from http://www.cacti.net/downloads/cacti-latest.tar.gz
2019-09-06_11:47 [Upgrade] Downloading Spine from http://www.cacti.net/downloads/spine/cacti-spine-latest.tar.gz
2019-09-06_11:47 [Upgrade] Installing new version of Cacti.
2019-09-06_11:47 [Restore] Setting cacti file permissions.
2019-09-06_11:47 [Upgrade] Compile + Installing new version of Spine.
./upgrade.sh: line 37: ./configure: No such file or directory
2019-09-06_11:47 [Upgrade] Copying config templates for Spine and Cacti.
2019-09-06_11:47 [Upgrade] Updating cacti/spine settings.
2019-09-06_11:47 [Upgrade] Attempting to update database via CLI.
You are attempting to install cacti 1.2.6 onto a 0.6.x database.
To continue, you must create a new database, import 'cacti.sql' into it,
and	update 'include/config.php' to point to the new database.
2019-09-06_11:47 [Upgrade] Cleaning temp files.
2019-09-06_11:47 [Upgrade] Upgrade complete, please log into cacti to finish.
```

**Comment by scline** [comment]
Looks like new versions of spine require `./bootstrap` to be ran. This will create the `./configure` file required to compile. 

bootstrap process requires `which` to be installed as well or it fails to locate `dos2unix`.

**Comment by mariusmotea** [comment]
Your image need to be fixed because is the best and simple cacti installation.

**Comment by scline** [comment]
https://github.com/scline/docker-cacti/pull/50 created with the required fixes for spine 1.2.6, will merge this weekend if no issues are found.

You can use and test this by using `smcline06/cacti:1.2.6`

**Comment by mariusmotea** [comment]
Thank you!
